packages/rest/src/rest.server.ts

```diff
@@ -23,9 +23,18 @@ import {Application, CoreBindings, Server} from '@loopback/core';
 import {getControllerSpec} from '@loopback/openapi-v3';
 import {HttpHandler} from './http-handler';
 import {DefaultSequence, SequenceHandler, SequenceFunction} from './sequence';
-import {FindRoute, InvokeMethod, Send, Reject, ParseParams} from './types';
+import {
+  FindRoute,
+  InvokeMethod,
+  Send,
+  Reject,
+  ParseParams,
+  Request,
+  Response,
+} from './types';
 import {RestBindings} from './keys';
-import {RequestContext} from '.';
+import {RequestContext} from './request-context';
+import * as express from 'express';
 
 export type HttpRequestListener = (
   req: ServerRequest,
```

Comment:
Should this be `Request` not `ServerRequest`?

Reply:
No. `HttpRequestListener` is passed to `http.createServer`, it must work with core `ServerRequest`/`ServerResponse` (no Express additions).